src/app/routes/getInitialData/utils/getRouteProps/index.js

```diff
@@ -0,0 +1,25 @@
+import { matchRoutes } from 'react-router-config';
+import pathOr from 'ramda/src/pathOr';
+import { fallbackAmpParam, fallbackServiceParam } from './routeFallbackParams';
+
+const getRouteProps = (routes, url) => {
+  const matchedRoutes = matchRoutes(routes, url);
+
+  const route = pathOr(undefined, [0, 'route'], matchedRoutes);
```

Comment:
you could just use ramda's `path` function because `undefined` is returned by default if the path is not found

` const route = path([0, 'route'], matchedRoutes);`